test/faker/default/test_faker_types.rb

```diff
@@ -69,6 +69,23 @@ def test_array_has_the_right_array
     assert @tester.rb_array.length == 1
   end
 
+  def test_rb_integer_array_has_the_right_array
+    assert @tester.rb_integer_array.length == 1
+    assert @tester.rb_integer_array(len: 3).length == 3
+    assert @tester.rb_integer_array(len: 0).empty?
+
+    if RUBY_VERSION < '2.4.0'
+      assert @tester.rb_integer_array.first.class == Fixnum
+    else
+      assert @tester.rb_integer_array.first.class == Integer
+    end
```

Comment:
Faker has been dropped support for Ruby 2.4.
https://github.com/faker-ruby/faker/pull/2038

Can you remove the condition branch?

```suggestion
    assert @tester.rb_integer_array.first.class == Integer
```